app/views/site/contribute.erb

```diff
@@ -0,0 +1,130 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Contributing Guide</h1>
+
+    <p class="lead">
+      <strong>So you want to contribute to exercism?</strong><br />
+      That is great news! We welcome all sort of contributions and will do everything we can to help you get started.
+    </p>
+
+    <p>
+      <strong>Exercism is a project comprised of several repositories and these can be divided basically in:</strong>
+
+      <ol>
+        <li><strong>The website</strong>: Its here that we can improve exercism features or layout.</li>
+        <li><strong>Command line interface</strong>: Fetch exercises and submit solutions.</li>
+        <li><strong>Language track repositories</strong>: Each language has its own repository with available problems.</li>
+        <li><strong>Problem metadata</strong>: Contains problem descriptions.</li>
+        <li><strong>Problems API</strong>: Delivers exercises, consumed by the Exercism command-line client, as well as the Exercism website.</li>
+        <li><strong>Rikki- the robot</strong>: Provides automated feedback on certain exercises.</li>
+      </ol>
+    </p>
+  </section>
+
+  <hr>
+
+  <section class="contributing-areas">
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Make Up Exercises</h2>
+
+          <p>
+            If you have an idea for an exercise you can go to the <a href="https://github.com/exercism/x-common">x-common</a> repository, there you will find existing exercises and can check if your exercise is not already defined.
+          </p>
+
+           <p>
+            You can check <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">here</a> to learn how to implement a new exercise.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Translate Exercises</h2>
+
+          <p>
+            Translating an exercise to your favorite language is a great way to help the comunity, track maintainers will be more than happy to give all the support needed so you can complete your first contribution.
+          </p>
+
+          <p>
+            A detailed guide can be seen <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track">here</a> if you want to give it a try!
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Review Pull Requests</h2>
+
```

Comment:
_We want the quality of the exercises to be as high as possible (and reviewing pull requests is a great way to both learn and teach a programming language)._

_We look at all sorts of things: style, idiomatic use of language features, naming, missing test cases, and redundant test cases. We also try to avoid forcing the solution to take a particular approach. Subscribe to the language track repositories of your favorite languages so you get notifications about new issues and pull requests. The track maintainers will greatly appreciate your help!_